src/fatx_internal.h

```diff
@@ -58,6 +58,7 @@
 /* Markers used in the filename_size field of the directory entry. */
 #define FATX_DELETED_FILE_MARKER     0xe5
 #define FATX_END_OF_DIR_MARKER       0xff
+#define FATX_IS_END_OF_DIR(marker)  ((marker == 0xff) || (marker == 0x00))
```

Comment:
* Let's `#define FATX_END_OF_DIR_MARKER2 0x00` and use the definitions.
* We should wrap `marker` in parenthesis
* Alignment

Reply:
Done